test/test_resource.rs

```diff
@@ -0,0 +1,22 @@
+use nix::sys::resource::{Resource, getrlimit, setrlimit};
+
+#[test]
+pub fn test_resource_limits() {
+    let mut limit = getrlimit(Resource::RLIMIT_STACK).unwrap();
+    assert!(limit.0 != limit.1);
```

Comment:
This assertion need not hold universally.  On FreeBSD for example, the default hard limits and soft limits seem to be identical.

Reply:
Done